mission-vision-values.md

```diff
@@ -0,0 +1,100 @@
+practice it’s not “just code” – there is also necessary alignment around
+protocols and semantic conventions for tags, attributes, and other metadata to
+consider.
+
+So how do we get high-quality, turnkey telemetry without a massive,
+unsustainable engineering effort? This is where OpenTelemetry comes in. To
+achieve our vision, OpenTelemetry sees five key opportunities, listed here:
+
+**Opportunity I: Telemetry should be easy.** With OpenTelemetry we want
+high-quality telemetry to be easy, especially for end-users. That means that
+OpenTelemetry should have fast time-to-value, set reasonable defaults yet allow
+for customization, and pair with excellent documentation and a top-tier overall
+developer experience.
+
+**Opportunity II: Telemetry should be universal.** Telemetry protocols and
+conventions should be unified across languages and signal types (tracing,
+metrics, logging, etc), not divergent or siloed. This means that OpenTelemetry
+aspires to find technical solutions that work consistently, both locally and
+globally.
+
+**Opportunity III: Telemetry should be vendor-neutral.** For decades, monitoring
+and observability vendors have created differentiation – and lock-in – through
+agents that automatically gather useful telemetry from a wide integration
+surface. This lock-in is limiting for customers and inhibits innovation by
+tightly coupling telemetry collection and telemetry storage and analysis. With
+OpenTelemetry, we strive to provide a level playing field for all observability
+providers, avoiding lock-in for any vendor and interoperating with other OSS
+projects in the telemetry and observability ecosystem.
+
+**Opportunity IV: Telemetry should be loosely coupled.** OpenTelemetry end-users
+should be able to pick and choose from the pieces they want without bringing in
+the rest of the project. To enable this, OpenTelemetry’s software architecture
+is decoupled wherever possible. As a corollary, this also means that
+OpenTelemetry does not want to “pick winners” when it comes to particular
+projects or technologies: where possible, we prefer to create optionality for
+our end-users.
+
+**Opportunity V: Telemetry should be built-in.** Historically, telemetry was
+something developers integrated manually or via post-compilation agents.
+OpenTelemetry believes that high-quality telemetry can be built in to the
+entire software stack – just like testing has been for decades.
+
+While the structure and technical details of OpenTelemetry may change over
+time, these five key opportunities will remain outstanding until we achieve our
+mission, and as a project we refer to them to orient – and reorient – as we
+chart our path.
+
+## Engineering Values (the principles that guide our contributions)
+
+OpenTelemetry’s mission and vision describe where we want to go.
+OpenTelemetry’s engineering values describe how we want to get there.
+
+OpenTelemetry’s core engineering values are Compatibility, Stability,
```

Comment:
I wish a user focus ("ease of use"? "usability"?) were a part of these engineering values

Reply:
"Telemetry should be easy" is the first part of the vision... is that sufficient?

Reply:
I'd say probably not - I think vision generally drives high level strategy / decision making while values are more for day-to-day. I hope engineers on the project value user experience in all their work and don't get that sense from the current wording. Of course this is just my hope - it's perfectly possible for my own values to not align with the project, it's up to the project to decide what to put here :P